tailcfg/tailcfg.go

```diff
@@ -1473,6 +1473,18 @@ func (pr PortRange) Contains(port uint16) bool {
 
 var PortRangeAny = PortRange{0, 65535}
 
+func (pr PortRange) String() string {
+	var buf bytes.Buffer
```

Comment:
https://pkg.go.dev/strings#Builder would be more efficient, but really you don't need either.

you want:

```go
	if pr.First == pr.Last {
		return strconv.FormatInt(pr.First, ...)
	} else if pr == PortRangeAny {
		return "*"
	} else {
		return fmt.Sprintf("%d-%d", pr.First, pr.Last)
	}
}